powerline/segments/i3wm.py

```diff
@@ -115,3 +115,37 @@ def mode(pl, segment_info, names={'default': None}):
 	if mode in names:
 		return names[mode]
 	return mode
+
+
+def scratchpad_groups(w):
+	group = []
+	if w.urgent:
+		group.append('s_urgent')
+	if w.nodes[0].focused:
+		group.append('s_focused')
+	if w.workspace().name != '__i3_scratch':
+		group.append('s_visible')
+	group.append('scratchpad')
+	return group
+
+
+def scratchpad(pl, fresh_icon=u'\u25cf', changed_icon=u'\u25cb'):
```

Comment:
Default icons must be ASCII. Unicode strings must not have `u` prefix, there is `from __future__ import unicode_strings`. They must also be placed into all theme files, even if theme defines them identical to default ones.